### ISSUE TITLE
torch.export converts all operator arguments to positional, losing parameter names - impacts MLIR lowering

### ISSUE THREAD

**Issue by LiFeiYu-ws** [pytorch/pytorch#166851]
### 🚀 The feature, motivation and pitch

Description
When exporting models using torch.export.export(), both built-in PyTorch operators and custom operators have their keyword arguments automatically converted to positional arguments in the exported graph. This results in the loss of parameter name information, which is critical for:
1. MLIR lowering: MLIR dialects rely on named attributes to correctly map operations
2. Semantic preservation: Parameter names provide essential semantic meaning that is lost with positional-only representation

Expected Behavior
Operators with named parameters should preserve those names as keyword arguments in the exported graph, enabling downstream tools to understand the semantic meaning of each parameter.

### Additional context

<img width="1139" height="373" alt="Image" src="https://github.com/user-attachments/assets/05cb04c9-e155-4f27-bfa1-9893a424edf0" />

cc @chauhang @penguinwu @avikchaudhuri @gmagogsfm @zhxchen17 @tugsbayasgalan @angelayi @suo @ydwu4

**Comment by angelayi** [comment]
Based on the schema of [aten.view.default](https://github.com/pytorch/pytorch/blob/68eb55c4b23babd005267dfd322dc4b070041f58/aten/src/ATen/native/native_functions.yaml#L8321), all arguments are positional and it has no kwargs. If we look at the schema for [aten.index_add.out](https://github.com/pytorch/pytorch/blob/68eb55c4b23babd005267dfd322dc4b070041f58/aten/src/ATen/native/native_functions.yaml#L8355) it has two kwargs, so I would expect the kwarg in the fx.Node to be filled. Therefore the behavior you noted seems to be expected.

You can run a pass and convert all the positional arguments to kwargs if you want. It would look something like:
```python
for node in graph.nodes:
    if node.op == "call_function" and isinstance(node.target, torch._ops.OpOverload):
        schema = node.target._schema
        kwargs = {}
        for i, arg in enumerate(schema.arguments):
            if arg.kwarg_only and arg.name in node.kwargs:
                kwargs[arg.name] = node.kwargs[arg.name]
            else:
                kwargs[arg.name] = node.args[i]

        node.args = ()
        node.kwargs = kwargs
```
